### DISCUSSION TITLE
how to deploy?

### DISCUSSION THREAD

**Discussion by text2sql** [mayooear/ai-pdf-chatbot-langchain#61] [Q&A]
So, i ve managed to get it run locally. but what is the next step to host it online? do i host the directory that i have on my machine after building all dependencies, or an initial version from Mayo (and then make changes on the platform)? sorry , i have almost zero software dev experience. thank you!

**Comment by text2sql** [comment]
figured it out. thank you!

**Reply by P-hongsi** [reply]
can u share to me?
